gum/arch-arm64/gumarm64writer.h

```diff
@@ -147,11 +101,11 @@ void gum_arm64_writer_put_sub_reg_reg_imm (GumArm64Writer * self,
 void gum_arm64_writer_put_nop (GumArm64Writer * self);
 void gum_arm64_writer_put_brk_imm (GumArm64Writer * self, guint16 imm);
 
-void gum_arm64_writer_put_push_all_registers(GumArm64Writer * self);
-void gum_arm64_writer_put_push_all_Q_registers(GumArm64Writer * self);
+void gum_arm64_writer_put_push_all_registers (GumArm64Writer * self);
+void gum_arm64_writer_put_push_all_q_registers (GumArm64Writer * self);
```

Comment:
I'd say `gum_arm64_writer_put_push_all_x_registers` or, to be more precise, `gum_arm64_writer_put_push_all_x_registers_and_nzcv`.